man/docker-create.1.md

```diff
@@ -158,6 +159,11 @@ two memory nodes.
                                'container:<name|id>': reuses another container shared memory, semaphores and message queues
                                'host': use the host shared memory,semaphores and message queues inside the container.  Note: the host mode gives the container full access to local shared memory and is therefore considered insecure.
 
+**--isolation**="*default*"
```

Comment:
Rather than trying to cover all this in the  flag, please keep the command simple and add the content you created for `create` above here in the man page.